viewing_party/party.py

```diff
@@ -1,23 +1,175 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
+    
+    movie_dict["title"] = title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating
+
+    return movie_dict
+
+
+def add_to_watched(user_data, movie):
+    watched = user_data["watched"] 
+    if watched is None:        
+        watched = [movie]
+    watched.append(movie)
+
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    watchlist = user_data["watchlist"]
+    if watchlist is None:
+        watchlist= [movie]

```

Comment:
Nitpick: Missing whitespace before `=`
```suggestion
        watchlist = [movie]

```